admin/taxes.md

```diff
@@ -26,10 +26,13 @@ If you use Avalara or TaxCloud to find a matching rate, that rate will override
 
 ![](/assets/admin-dashboard-taxes-avalara.png "Avalara Configuration")
```

Comment:
I started to but decided we should wait until all those admin shots get updated.